_pages/en/mandate.md

```diff
@@ -8,40 +8,52 @@ sections: Mission
 permalink: /mandate.html
 ---
 
-## Vision (image of the future)
+- Diagnose key organizational problems, waste, lead times and governance structures that inhibit the desired changes in behaviours
+- Recommend specific course of actions to governance bodies to adopt new methods of managing IT risks
+- Measure progress of adoption of strategies in IT teams and governance bodies as a performance indicator of the IT Strategy team
+- Challenge current thinking and approaches
 
-The [Canadian Experience](human-development-life-cycle.html) is when Canadians are benefiting seamlessly of the federal government services through modern technologies.
+<img src="assets/images/our-values.png" alt="">
+- Learning and Continuous Improvement
+- Self-Discipline
```

Comment:
I can't put it into words well but 'Self-Discipline' as a value leaves me with an unpleasant feeling. It feels...judgemental.  Curious what the thinking was behind that one.

Reply:
Might not be the best word but I mean "we take responsibility for managing our workload and  participate in team decision making to come up with a solution to any problem statement"